libraries/AP_Landing/AP_Landing_Deepstall.cpp

```diff
@@ -255,10 +262,20 @@ bool AP_Landing_Deepstall::verify_land(const Location &prev_WP_loc, Location &ne
         Location entry_point;
         landing.nav_controller->update_waypoint(arc_exit, extended_approach);
 
-        float relative_alt_D;
-        landing.ahrs.get_relative_position_D_home(relative_alt_D);
+        float height_above_target;
+        if (is_zero(approach_alt_offset)) {
+            landing.ahrs.get_relative_position_D_home(height_above_target);
+            height_above_target = -height_above_target;
+        } else {
+            Location position;
+            if (landing.ahrs.get_position(position)) {
+                height_above_target = (position.alt - landing_point.alt + approach_alt_offset * 100) / 100.0;
```

Comment:
maybe *0.01f ;-P

Reply:
YES! Definitely definitely yes!

Reply:
You know I saw it when I sent in the actual PR and figured I'd see if anyone else would notice :D